modules/ossm-release-2-6-2.adoc

```diff
@@ -7,7 +7,7 @@ Module included in the following assemblies:
 [id="ossm-release-2-6-2_{context}"]
 = {SMProductName} version 2.6.2
 
-This release of {SMProductName} includes 2.6.2, 2.5.5, and 2.4.11; addresses Common Vulnerabilities and Exposures (CVEs); contains bug fixes; and is supported on {product-title} 4.14 and later.
+This release of {SMProductName} includes 2.6.2, 2.5.5, and 2.4.11; addresses Common Vulnerabilities and Exposures (CVEs); contains bug fixes; and is supported on {product-title} 4.14 and later. This version is included in {SMProductName} Operator 2.6.2.
```

Comment:
Hrm... 2.6.2 is the operator ... maybe... something like this:

This release of {SMProductName} updates the {SMProductName} Operator version to 2.6.2, and includes the following `ServiceMeshControlPlane` resource version updates: 2.5.5 and 2.4.11. 

This release addresses Common Vulnerabilities and Exposures (CVEs); contains bug fixes; and is supported on {product-title} 4.14 and later.

@mkralik3 , thoughts?